etna/transforms/math/lags.py

```diff
@@ -273,7 +287,10 @@ def _transform(self, df: pd.DataFrame) -> pd.DataFrame:
             feature = df.loc[:, pd.IndexSlice[:, feature_name]]
 
             if shift > 0:
-                shifted_feature = feature.shift(shift, freq=self._freq)
+                # TODO: разобраться с этим моментом и преобразованием типов
+                # shifted_feature = feature.shift(shift, freq=self._freq)
+                # shifted_feature = shifted_feature.loc[shifted_feature.index.intersection(df.index)]
```

Comment:
I think it is better to write a more comprehensive comment and remove code samples.